extensions/dicom-microscopy/src/DicomMicroscopyANNSopClassHandler.js

```diff
@@ -0,0 +1,103 @@
+import OHIF, { DicomMetadataStore } from '@ohif/core';
+import loadAnnotation from './utils/loadAnnotation';
+import getSourceDisplaySet from './utils/getSourceDisplaySet';
+
+const { utils } = OHIF;
+
+const SOP_CLASS_UIDS = {
+  MICROSCOPY_BULK_SIMPLE_ANNOTATION: '1.2.840.10008.5.1.4.1.1.91.1',
+};
+
+const SOPClassHandlerId =
+  '@ohif/extension-dicom-microscopy.sopClassHandlerModule.DicomMicroscopyANNSopClassHandler';
+
+function _getDisplaySetsFromSeries(instances, servicesManager, extensionManager) {
+  // If the series has no instances, stop here
+  if (!instances || !instances.length) {
+    throw new Error('No instances were provided');
+  }
+
+  const { displaySetService, microscopyService } = servicesManager.services;
+
+  const instance = instances[0];
+
+  const naturalizedDataset = DicomMetadataStore.getSeries(
+    instance.StudyInstanceUID,
+    instance.SeriesInstanceUID
+  ).instances[0];
+
+  const {
+    SeriesDescription,
+    ContentDate,
+    ContentTime,
+    SeriesNumber,
+    StudyInstanceUID,
+    SeriesInstanceUID,
+    SOPInstanceUID,
+    SOPClassUID,
+  } = instance;
+
+  const displaySet = {
```

Comment:
Can you add:
    isOverlayDisplaySet: true,

and then use that to tell if this display set is one that is overlay for the microscopy viewer.

Reply:
Not sure why we need this? I'm not using a cornerstone viewport.

Reply:
It means that you don't need to test for hte modality of a display set, but just do:
ds.isOverlayDisplaySet
That is, it is the display set that determines whether it is handled as an overlay or not.

Reply:
Ok but where isOverlayDisplaySet is being consumed/used for microscopy?

Reply:
You would use it instead of doing the modality===sr || modality===ann so that the display set itself can decide whether it is an overlay.

Reply:
Updated.